From 8c4a310aa3a44223c99d356bdafe823051c5bf3d Mon Sep 17 00:00:00 2001
From: Paulo Matias <matias@ufscar.br>
Date: Sat, 6 Jun 2020 09:39:45 -0300
Subject: [PATCH] Fix incomplete build type error
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

```
In file included from /home/main-builder/pkgwork/src/hal-git/src/gui/graph_widget/graph_layout_progress_widget.cp
p:1:
/home/main-builder/pkgwork/src/hal-git/include/gui/graph_widget/graph_layout_progress_widget.h:44:18: error: fiel
d ‘m_arrow_left’ has incomplete type ‘QPainterPath’
   44 |     QPainterPath m_arrow_left;
      |                  ^~~~~~~~~~~~
```
---
 include/gui/graph_widget/graph_layout_progress_widget.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/gui/graph_widget/graph_layout_progress_widget.h b/include/gui/graph_widget/graph_layout_progress_widget.h
index ca985111..ef167108 100644
--- a/include/gui/graph_widget/graph_layout_progress_widget.h
+++ b/include/gui/graph_widget/graph_layout_progress_widget.h
@@ -2,6 +2,7 @@
 #define GRAPH_LAYOUT_PROGRESS_WIDGET_H
 
 #include <QFrame>
+#include <QPainterPath>
 
 class QParallelAnimationGroup;
 class QPropertyAnimation;
-- 
2.27.0

